src/app/query/stacks/transactions/transactions-with-transfers.hooks.ts

```diff
@@ -0,0 +1,12 @@
+import { useMemo } from 'react';
+
+import { useGetAccountTransactionsWithTransfers } from './transactions-with-transfers.query';
+
+export function useStacksTransactionsWithTransfers() {
+  return useGetAccountTransactionsWithTransfers().data?.results ?? [];
+}
```

Comment:
Can we use the hook directly, rather than creating this wrapper? Or, how might we make it easy to later consume the other properties of `react-query`?

Custom `onSelect`? 🤔

Reply:
I'll just use the hook directly, and refactor others I come across from now on.